lib/schemaUtils.js

```diff
@@ -2018,6 +2019,10 @@ module.exports = {
                 REQUEST_TYPE.ROOT, PARAMETER_SOURCE.REQUEST, components, options, schemaCache));
             }
           });
+
+          if (encoding[key] && encoding[key].hasOwnProperty('contentType')) {
```

Comment:
Can use `_.get(encoding, [${key}]contentType)` to make sure to avoid type errors in cases where `encoding[key]` is defined `null`.